examples/multi-destination-content.adoc

```diff
@@ -50,6 +50,8 @@ Some overview
 ----
 
 ifdef::backend-revealjs[=== !]
+// TODO one day would be nice to allow
+//ifdef::backend-reveal.js[=== !]
```

Comment:
This should already work. The `.` is legal in this context.

Reply:
But it will only be true if you activate the converter using `-b reveal.js` instead of `-b revealjs`. So it's either or. The nice part would be "and".